README.md

```diff
@@ -1,74 +1,108 @@
 ![](docs/lift.png)
 
-*Shipping Serverless features made easy*
+Lift is a plugin that leverages the AWS CDK to expand the [Serverless Framework](https://www.serverless.com/) beyond functions.
 
-## Features
+Deploy production-ready websites, queues, storage buckets and more with a few lines in serverless.yml.
 
-Lift is a [Serverless](https://www.serverless.com/) plugin that simplifies deploying well-architected serverless applications.
-
-It is made for developers new to serverless, as well as experienced AWS builders.
-
-- ⚡️ **Get started with Serverless** - Developer-friendly vocabulary, no AWS knowledge required
-- 📦 **Faster delivery** - Production-ready components with minimal configuration
-- 🔁 **Not invasive** - Integrates with existing Serverless Framework projects
-- ⏏️ **No lock-in** - Eject to CloudFormation at any time
+- ⚡️ **For developers** - No AWS knowledge required
+- ⚡️ **Production-ready** - Built by AWS experts, optimized for production
+- ⚡️ **Not invasive** - Integrates with existing projects
+- ⚡️ **No lock-in** - Eject to CloudFormation at any time
 
 [Why should I choose Lift ?](docs/comparison.md)
 
 ## Installation
 
+Lift is a [Serverless Framework plugin](https://www.serverless.com/plugins/), install it in your project:
+
 ```bash
-npm i -D serverless-lift
+serverless plugin install -n serverless-lift
 ```
 
+If you prefer, you can install via `npm` directly: `npm i -D serverless-lift`.
```

Comment:
I always find the short syntax confusing, especially for non JS devs, which serverless framework users may not be.

And most of the time it would be copy-pasted anyway.

```suggestion
If you prefer, you can install via `npm` directly: `npm install --save-dev serverless-lift`.
```

Reply:
Good point. I wasn't familiar with JS for a long time and now that you mention it, I remember being confused/worried with these unknown options.